docs/modders/Bonus/Bonus_Types.md

```diff
@@ -1023,3 +1023,7 @@ Internal bonus, do not use
 ### DISINTEGRATE
 
 After death of unit no corpse remains
+
+### INVINCIBLE
+
+Cannot be target of attacks and spells and cannot be affected by anything
```

Comment:
```suggestion
The unit affected by this bonus cannot be target of attacks or spells and cannot be affected by anything.
```
BTW - what is 'anything'? Attacks and spells are already covered. Perhaps remove?